### ISSUE TITLE
Special chars in From and Subject

### ISSUE THREAD

**Issue by caramboleyo** [EC-Nordbund/denomailer#36]
When you use special chars in subject or from in some email clients they are not displayed properly.

I checked an email send with rainloop, where it works, and they seem to escape words containing special chars like this: 
"=?utf-8?B?w5ZHS1YgS8Okcm50ZW4=?="

Meanwhile is there an escape function to archive this right now? When i copied that string into my send function it was displayed correctly.

**Comment by mathe42** [comment]
Could you provide a minimal code example?

**Comment by caramboleyo** [comment]
Some mail clients now show ??? instead of `Ö Ä Ü ß ö ä ü`

```js
import { SMTPClient } from "https://deno.land/x/denomailer/mod.ts";

const client = new SMTPClient({
    connection: {
        hostname: 'smtp.mailbox.org',
        port: 465,
        tls: true,
        auth: {
            username: 'xxx',
            password: 'xxx',
        },
    },
});
console.log('>>>send' , await client.send({
    from: 'Önsen Stärebrü <oensen.staerebrue@example.com>',
    to: 'Inga Wäregwö <recipient@example.com>',
    subject: 'Köttbullar recipe by Önsen Stärebrü',
    html: '<html><head></head><body>This <b>is</b> a test.</body></html>',
}));
client.close();
```

**Comment by mathe42** [comment]
You can try 

```ts
import { quotedPrintableEncode } from "https://deno.land/x/denomailer/config/mail/encoding.ts";

const newSubject = quotedPrintableEncode(oldSubject)
```

might work.

**Comment by caramboleyo** [comment]
No, that looks like this:
K=c3=b6ttbullar recipe by =c3=96nsen St=c3=a4rebr=c3=bc

They seem to wrap it inside `=?utf-8?B? .. ?=` and hash it somehow.

**Comment by mathe42** [comment]
It looks like you can do
```ts
const newSubject = `=?utf-8?Q?${quotedPrintableEncode(oldSubject)}?=`
```

see https://www.telemessage.com/developer/faq/how-do-i-encode-non-ascii-characters-in-an-email-subject-line/

I can't reproduce the error with 3 different SMTP-Servers so not sure what to expect...

**Comment by mathe42** [comment]
I created a PR that implements the above fix.

You can try it by importing denomailer from `https://raw.githubusercontent.com/EC-Nordbund/denomailer/feat/encode-mail-name%2Bsubject/mod.ts`

**Comment by caramboleyo** [comment]
Yes, with the new mod.ts file it works out of the box. 👏👏👏

**Comment by mathe42** [comment]
It semes to work (before this fix) with non spec-compliant-clients so this is an important change!

**Comment by mathe42** [comment]
Released in version 1.3.0 thanks a lot for the help!

https://deno.land/x/denomailer@1.3.0/mod.ts

**Comment by nestarz** [comment]
Hi @mathe42 , When I have ascii chars in my subject it shows weird tags in my email
![image](https://github.com/EC-Nordbund/denomailer/assets/5360876/debe0e5d-17b6-4202-ac35-dfc10a51878b)

Is it linked ?
